claat/cmd/update.go

```diff
@@ -31,7 +31,10 @@ import (
 )
 
 // CmdUpdate is the "claat update ..." subcommand.
-func CmdUpdate() {
+// prefix is a URL prefix to prepend when using HTML format.
+// globalGA is the global Google Analytics account to use.
+// authToken is the token to use for the Drive API.
+func CmdUpdate(prefix, globalGA, authToken string) {
```

Comment:
This is more of a Golang stylistic thing, but I'd like to get your opinion and others. When a function exceeds 2 arguments, I usually find it more helpful to accept a struct:

```golang
type CmdUpdateInput {
  Prefix string
  // ...
}

func CmdUpdate(i *CmdUpdateInput) {
  // i.Prefix, i.GlobalGA, i.AuthToken
}
```

This helps prevent future breaking API changes. For example, if that pattern were already adopted here, you could have added `authtoken` to the struct without changing the existing API. Thoughts?

Reply:
I like that idea -- I'll use structs instead.